internal/pkg/manifest/validate_test.go

```diff
@@ -305,6 +305,26 @@ func TestLoadBalancedWebService_Validate(t *testing.T) {
 			},
 			wantedError: errors.New(`scaling based on "nlb" requests or response time is not supported`),
 		},
+		"error if fail to validate Deployment configuration": {
```

Comment:
```suggestion
		"error if fail to validate deployment": {
```
and same for the other places 😛